libs/search/src/lib/model.ts

```diff
@@ -7,6 +7,9 @@ export interface RecordSimple {
   abstract: string
   url: string
   thumbnailUrl: string
+  category: string
+  more: string
```

Comment:
What is `more`, what is `open` ?
These are properties of the metadata itself, there is a miss understanding here.

Reply:
I removed those, they should be gone